snap/snapcraft.yaml

```diff
@@ -0,0 +1,26 @@
+name: firecracker
+version: git
+summary: Secure and fast microVMs.
+description:| 
+Secure and fast microVMs for serverless computing.
+  
+grade: stable
+confinement: strict
+base: core18
+
+architectures:
+  - build-on: amd64
+  - build-on: armhf
+  - build-on: arm64
+  
+parts:
+  firecracker:
+    plugin: rust
```

Comment:
This will use latest rust, you should specify rust version 1.32 using `rust-revision`.

Reply:
It seems that at some point rustup.sh removed the --prefix option, favoring the environment variables, and this needs to be fixed on the snapcraft side of things: https://github.com/rust-lang/rustup.rs/issues/1703.